File: app/src/main/java/com/example/qiuchenly/comicparse/Adapter/BaseFragmentPagerStatement.java
package com.example.qiuchenly.comicparse.Adapter;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentStatePagerAdapter;

import java.util.List;

public class BaseFragmentPagerStatement extends FragmentStatePagerAdapter {

    private List<Fragment> _fragment;

    public BaseFragmentPagerStatement(FragmentManager fm, List<Fragment> _fragment) {
        super(fm);
        this._fragment = _fragment;
    }

    @Override
    public Fragment getItem(int position) {
        return _fragment.get(position);
    }

    @Override
    public int getCount() {
        return _fragment.size();
    }
}

File: ComicX/src/main/java/com/qiuchenly/comicparse/MVP/UI/Adapter/BaseFragmentPagerStatement.kt
package com.qiuchenly.comicparse.MVP.UI.Adapter

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter

class BaseFragmentPagerStatement(fm: FragmentManager, private val _fragment: List<Fragment>) : FragmentStatePagerAdapter(fm) {

    override fun getItem(position: Int): Fragment {
        return _fragment[position]
    }

    override fun getCount(): Int {
        return _fragment.size
    }
}
